packages/lwc/types.d.ts

```diff
@@ -4,6 +4,9 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
+
+import { HTMLElement } from '@lwc/template-compiler/dist/types/shared/types';
```

Comment:
Rather than import the `HTMLElement` type definition from template-compiler, we should stick to the same definition source for all the DOM-related type definitions.

I believe that should be typescript's `lib.dom.t.s`. I'm not sure what the best way to pull that in here...it seems to be sufficient to create an empty `tsconfig.json` file. Could we remove this line for now and we can figure out the right way to do it in a separate PR? /cc @pmdartus

Reply:
Looked into this a little more and I believe that https://github.com/salesforce/lwc/pull/1993 is what we need.

Reply:
ok, thanks, I can remove line 8.